khelo/lib/ui/flow/stats/user_stat/user_stat_view_model.dart

```diff
@@ -9,7 +9,8 @@ import 'package:freezed_annotation/freezed_annotation.dart';
 part 'user_stat_view_model.freezed.dart';
 
 final userStatViewStateProvider =
-    StateNotifierProvider<UserStatViewNotifier, UserStatViewState>((ref) {
+    StateNotifierProvider.autoDispose<UserStatViewNotifier, UserStatViewState>(
```

Comment:
can you explain behaviour of stream as we have used autoDispose, won't it dispose and reinit streamSubscription each time we change the tab? isn't it increasing firebase query count?